changelogs/fragments/1753-document-fstypes-supported-by-resizefs.yml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
A docs-only PR doesn't need a changelog fragment. If you want to keep the info on the `filesystem` module, that's ok, but please remove the part on the `lvol` module.